sdk/include/psdk/imm.h

```diff
@@ -270,23 +270,6 @@ BOOL WINAPI
 ImmDestroySoftKeyboard(
     _In_ HWND hwndSoftKBD);
 
-BOOL WINAPI ImeInquire(LPIMEINFO, LPWSTR, LPCWSTR lpszOptions);
```

Comment:
You confirm that those "Ime*" functions are indeed not present in the official MS PSDK imm.h file?

Reply:
Yep, they should be in `immdev.h`.